ckanext/xloader/plugin.py

```diff
@@ -206,4 +206,5 @@ def get_helpers(self):
             'xloader_status': xloader_helpers.xloader_status,
             'xloader_status_description':
             xloader_helpers.xloader_status_description,
+            'is_xloader_format': xloader_helpers.is_xloader_format,
```

Comment:
you could skip the extra function in helpers and

```suggestion
            'is_xloader_format': XLoaderFormats.is_it_an_xloader_format,
```
instead. But either way is fine (leaving it in helpers would let us use the blanket stuff in the future and remove get_helpers here)